cmd/machine.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"os"
 
+	packet_logr "github.com/packethost/pkg/log/logr"
 	"github.com/spf13/cobra"
+	v1 "github.com/tinkerbell/pbnj/api/v1"
```

Comment:
hmm I feel like this is unfortunate like this. I wonder how we can make this better in the future.